### ISSUE TITLE
digestif as provided by texlive does not seem to work

### ISSUE THREAD

**Issue by callegar** [astoff/digestif#53]
I do not know if the digestif upstream is directly involved in the texlive packaging or if they can help, if this is not the case, then my apology for the noise.

From the documentation, it is not clear to me how digestif is meant to be used in this scenario.

TeXLive makes a `digestif` executable available, however trying to invoke it invariably returns
```
/usr/local/texlive/2022/bin/x86_64-linux/digestif:8: Can't find 'digestif.zip' archive
```

Trying to use ESC-x eglot in emacs produces a `server died` error.

Any clue?

**Comment by dr-scsi** [comment]
I can confirm this also on Windows -- using the provided script (`digestif.cmd`) and a local git repo works as expected.

**Comment by zepinglee** [comment]
I can also reproduce this issue on my TeX Live 2022 with all packages up-to-date on macOS 12.6.

```
$ type digestif
digestif is /usr/local/texlive/2022/bin/universal-darwin/digestif
$ digestif --help
/usr/local/texlive/2022/bin/universal-darwin/digestif:8: Can't find 'digestif.zip' archive
```

It seem that `digestif.zip` archive is in the `TEXMF/scripts/digestif` directory but `kpse.find_file("digestif.zip")` cannot find it. This can be checked with kpsewhich executable.

I suggest putting `digestif.zip` in the `TEXMF/tex/generic/digestif` directory and it works fine.
```
$ kpsewhich digestif.zip
/Users/zepinglee/Library/texmf/tex/generic/digestif/digestif.zip
$ digestif --version
Digestif 0.5
```

**Comment by astoff** [comment]
I've submitted a fix in version 0.5.1.  It should be on TeXLive in a couple of days.

Thanks a lot for checking, and let me know if something else looks awry!

**Comment by callegar** [comment]
Thank you for the quick response! Looking forward to trying digestif!
